packages/fuels/src/cli/commands/init/index.ts

```diff
@@ -37,29 +40,41 @@ export function init(program: Command) {
   const noneIsInformed = ![workspace, contracts, scripts, predicates].find((v) => v !== undefined);
 
   if (noneIsInformed) {
-    // mimicking commander property validation
-    process.stdout.write(`error: required option '-w, --workspace <path>' not specified\r`);
+    const errorString = 'error: ';
+    const errorStringPadding = errorString.replaceAll(/./g, ' ');
+
+    const workspaceOption = (program as CommandWithInternals).options.find((o) => o.short === '-w')
+      ?.flags;
+    const optionsConflictingWithWorkspaceOption = (program as CommandWithInternals).options
+      .filter((o) => o.conflictsWith.includes('workspace'))
+      .map((o) => `'${o.flags}'`)
+      .join(', ');
+    // eslint-disable-next-line no-console
+    console.log(
```

Comment:
I changed `process.stdout.write` to `console.log` and then the message appeared correctly on the terminal. When `process.stdout.write` is used, the message is shown and then removed right after `process.exit(1)` is called. To test this out, you can do the following:
```ts
process.stdout.write('whatever');
setTimeout(() => process.exit(1), 1000);
```

I couldn't pinpoint the reason for this behavior. It might be due to `process.stdout` being a `WritableStream` which is an object existing as long as `process` exists and the message gets removed when `process` is killed... I'm not sure.

Reply:
Great find about the `console.log` vs `process.stdout`!

I appreciate the attention to error handling. However, I'm in favor of just replacing `process.stdout.write()` calls with `console.log()` and calling it a day. 

I don't think the added complexity justifies itself; besides, option validation and error reporting should ideally stay the same across all CLI options, which is why it was simply mimicking how `Commander` reports these problems.

Reply:
The original error message didn't consider the scenario that you can also pass `-c`/`-s`/`-p` and made it look like `-w` was the only way to go and that's why I went about it - for correctness' sake. I suppose the user would also see those flags via `pnpm fuels init --help` and via our documentation, though, so it's not _that_ big of an issue to leave the message as it was. I'm curious to hear what others think of this. @FuelLabs/sdk-ts

Reply:
> could this be a wider problem?

It definitely could be. The `_exit` override in the linked PR seems unnecessary given that we can use `console.log/warn/debug` instead.

Reply:
> I appreciate the attention to error handling. However, I'm in favor of just replacing process.stdout.write() calls with console.log() and calling it a day.

I went with this to simplify the discussion since the PR increased in scope.